nixos/modules/services/misc/paperless-ng.nix

```diff
@@ -61,6 +60,8 @@ let
     ProtectKernelModules = true;
     ProtectKernelTunables = true;
     ProtectProc = "invisible";
+    # django-q reads /proc/stat for CPU and memory information
+    ProcSubset = "all";
```

Comment:
Not important, but moving the option breaks the alphabetical sorting. (I see that you moved it to the other /proc option. Just wanted to mention the intention behind the original order.)

Reply:
Yes, I intentionally grouped these closely related options. We already do this with the file system options at the top whose order is not strictly alphabetical.